guesttemplates/blank_template.py

```diff
@@ -271,7 +271,7 @@ def toxml(self, version):
         struct = doc.createElement('struct')
         ver_member.appendChild(struct)
 
-        for n, v in version.items():
+        for n, v in list(version.items()):
```

Comment:
Why is this needed? `version` is a dict and it should be fine to iterate over the dict_view returned by items() as the loop body doesn't modify the dict.

Reply:
Mark commented about the same. It's futurize being conservative by default and converting what used to be a list to a list in python3, just in case we would have used properties of a list that a view doesn't have, such as addressing by index or being a copy you can modify without altering the dict. I'll check that these properties are not used.